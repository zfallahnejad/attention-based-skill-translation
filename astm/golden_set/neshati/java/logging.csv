1446916,logging
870248,logging
100970,logging
605744,logging
21234,logging
592139,logging
70604,logging
1831293,logging
438154,logging
571407,logging
