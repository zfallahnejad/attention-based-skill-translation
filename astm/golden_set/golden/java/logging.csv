1446916,33,35,0.4852941176470588
100970,25,28,0.4716981132075472
870248,25,28,0.4716981132075472
605744,24,14,0.631578947368421
21234,14,9,0.6086956521739131
70604,13,10,0.5652173913043478
592139,13,8,0.6190476190476191
1831293,12,5,0.7058823529411765
571407,11,3,0.7857142857142857
438154,11,5,0.6875
152794,10,5,0.6666666666666666
